lib/awful/menu.lua

```diff
@@ -140,6 +140,8 @@ local function load_theme(a, b)
     ret.width = a.width or b.menu_width or b.width or
                 fallback.menu_width or dpi(100)
     ret.font = a.font or b.font or fallback.menu_font or fallback.font
+    ret.opacity_disabled = a.opacity_disabled or b.menu_opacity_disabled or b.opacity_disabled or
+                           fallback.menu_opacity_disabled or 0.5
```

Comment:
I'm not sure how this widget works. Should we also have a new `beautiful` variable for this?